gamemode/modules/inventory/cl_itemtooltip.lua

```diff
@@ -0,0 +1,144 @@
+RVR.Inventory = RVR.Inventory or {}
+local inv = RVR.Inventory
+
+local PANEL = {}
+
+local h = ScrH() * 0.12
+local w = h * ( 921 / 261 ) * 1.6
```

Comment:
what are these magic numbers

Reply:
I removed 2 of them, the other two are basically unavoidable though